Calculator/Calculator/Model/LinkedList.swift

```diff
@@ -0,0 +1,31 @@
+struct LinkedList<Item: CalculatorItem> {
+    private var firstNode: Node<Item>?
+    private(set) var lastNode: Node<Item>?
+    var isEmpty: Bool {
+        return firstNode == nil
+    }
+    
+    mutating func add(value: Item) {
+        let newNode = Node(value)
+        
+        if isEmpty {
+            firstNode = newNode
+            lastNode = newNode
+        }else {
```

Comment:
```suggestion
        } else {
```
=> 아시겠지만 여기 한 칸 띄워주세요~ :)

Reply:
알고 있는 컨벤션인데 놓친 부분이 있었네요..!